sonic_platform_base/sfp_base.py

```diff
@@ -384,4 +419,10 @@ def write_eeprom(self, offset, num_bytes, write_buffer):
         """
         raise NotImplementedError
 
+    def get_error_description(self):
+        """
+        Retrives the error status of the SFP module
 
+        Returns:
```

Comment:
Could you please add description for the return value? I suppose if multiple error exists at the same time, "|" should be used to join them here.

Reply:
Added